dataactvalidator/config/sqlrules/fabs37_detached_award_financial_assistance_3.sql

```diff
@@ -0,0 +1,24 @@
+-- For new assistance awards (ActionType = A), the CFDA_Number must have been registered
+WITH detached_award_financial_assistance_fabs37_1_{0} AS
+    (SELECT submission_id,
+        row_number,
+        cfda_number,
+        action_type,
+        correction_late_delete_ind,
+        action_date
+    FROM detached_award_financial_assistance
+    WHERE submission_id = {0})
+SELECT
+    row_number,
+    cfda_number,
+    action_type,
+    correction_late_delete_ind,
+    action_date
```

Comment:
Why are we returning action_type, correction_late_delete_ind, or action_date if it's irrelevant for this rule? (same question for the WITH above it)